.travis.yml

```diff
@@ -51,4 +51,4 @@ jobs:
         - mv ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini{.disabled,}
         - if [[ ! $(php -m | grep -si xdebug) ]]; then echo "xdebug required for mutation tests"; exit 1; fi
       script:
-        - ./vendor/bin/infection --test-framework-options='--testsuite=unit' --threads=4 --min-msi=8 --min-covered-msi=100
+        - ./vendor/bin/infection --test-framework-options='--testsuite=unit' --threads=$(nproc) --min-msi=7 --min-covered-msi=100
```

Comment:
Why lower the `min-msi`?

Reply:
Because adding `(array)` increases the number of generated mutations and we have no tests covering this area. Probably using `assert()` is a more sensible solution but I'd delagate that refactor to #5.